etc/aqd.conf.noms

```diff
@@ -19,6 +23,18 @@ user_list_location =
 
 ###############################################################################
 
+[database]
+#database_section = database_postgresql
+database_section = database_sqlite
+
+[database_postgresql]
+dbname = aquilon
+
+[database_sqlite]
+dbfile = %(basedir)s/aquilon/db/aquilon.db
```

Comment:
As this is a file for community rather than MS, i do not mind. Just not sure if having a ‘db’ subdirectory in ‘aquilon’ directory was intentional or a typo that turned ‘aquilondb’ into to ‘aquilon/db’ by accident?